README.md

```diff
@@ -42,20 +42,22 @@ Many of the methods provided by this package have names similar to those in [R](
 Ordinary Least Squares Regression:
 ```julia
 julia> data = DataFrame(X=[1,2,3], Y=[2,4,7])
-3x2 DataFrame
-|-------|---|---|
-| Row # | X | Y |
-| 1     | 1 | 2 |
-| 2     | 2 | 4 |
-| 3     | 3 | 7 |
+3×2 DataFrames.DataFrame
```

Comment:
The `DataFrames.` prefix is going to go away in 0.7, so maybe better not add it now? Same for other prefixes of exported types (those available without the prefix).

Reply:
GLM.jl as well as Distributions.jl currently fails to run on Julia 0.7. Let us then leave this PR open then and I will try first to update them (maybe not all deprecations but at least errors).

Reply:
All needed changes for Julia 0.7 seem to be already submitted